### PR TITLE
Search pokemon

### PR DESCRIPTION
**Pull request by hania-jas**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -3,6 +3,9 @@ const pokeImg = document.querySelector('.pokeImg');
 const infoList = document.querySelector('.infoList');
 const pokeNameParagraph = document.querySelector('.pokeNameParagraph');
 const displayDateAndTimeParagraph = document.querySelector('.displayDateAndTimeParagraph');
+const input = document.querySelector('.input');
+
+let pokemonList = [];
 
 const displayDateAndTime = () => {
```
Comment (juwik) [inline script.js:10]: I would avoid blocks of code that big - I believe You should place some new lines to separate logic blocks inside that function (i.e. between ln 52 and 53).
Ideally, every "logic block" should be a separate function ;)

File: script.js [hunk 2]
```diff
@@ -41,35 +44,66 @@ const displayPhotoAndInfo = (pokemonDetailsUrl) => {
...
             pokeImg.appendChild(triangleRight);
+            const leftTriangleBackground = createShapes("DIV", 'leftTriangleBackground');
+            const rightTriangleBackground = createShapes("DIV", 'rightTriangleBackground');
+            pokeImg.appendChild(leftTriangleBackground);
+            pokeImg.appendChild(rightTriangleBackground);
             triangleRight.addEventListener('click', () => displayNextPokemon(pokeInfos.id));
             triangleLeft.addEventListener('click', () => displayPreviousPokemon(pokeInfos.id));
 
             pokeNameParagraph.innerText = pokeInfos.name;
+            pokeNameParagraph.style.background = 'linear-gradient(to top, #4ac329, #62e637, #22ac07)';
```
Comment (juwik) [inline script.js]: Please avoid these "inline stylings". It is better to give a proper CSS class.

File: script.js [hunk 3]
```diff
@@ -41,35 +44,66 @@ const displayPhotoAndInfo = (pokemonDetailsUrl) => {
...
+            pokeNameParagraph.style.border = '4px ridge #209f07';
             const pokemonInfoComponent = createPokemonInfo(pokeInfos);
             infoList.appendChild(pokemonInfoComponent);
-
-
         });
 }
+const searchPokemon = () => {
+    let inputValue = input.value;
```
Comment (juwik) [inline script.js]: Can be `const`

File: style.css [hunk 4]
```diff
@@ -1,134 +1,430 @@
...
-                top: 520px;
-                left: 820px;
-                position: fixed;
-                width: 190px;
-                height: 40px;
-                color: #7bd573;
-            }
\ No newline at end of file
+                @use postcss-color-function;
+                @use postcss-nested;
```
Comment (juwik) [inline style.css]: Are these `@use` necessary? What are these?

File: style.css [hunk 5]
```diff
@@ -1,134 +1,430 @@
...
+                    overflow: scroll;
+                    scrollbar-width: none;
+                    -ms-overflow-style: none;
+                }
+                
+                .listElement {
+                    margin: 0;
+                    padding: 15px 30px;
+                    list-style: none;
+                    background-image: url(https://img.icons8.com/color/48/000000/pikachu-pokemon.png);
```
Comment (juwik) [inline style.css]: Please download file and use it locally instead of urls

File: script.js (hunk 1 above)
Comment (hania-jas) [reply script.js:10]: Done! :)

File: script.js (hunk 2 above)
Comment (hania-jas) [reply script.js]: Fixed

File: script.js (hunk 3 above)
Comment (hania-jas) [reply script.js]: Fixed

File: style.css (hunk 4 above)
Comment (hania-jas) [reply style.css]: Fixed

File: style.css (hunk 5 above)
Comment (hania-jas) [reply style.css]: Fixed!

**Pull request merged**